dlt-database/migrations/0003-refactor_transaction_recipe.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  // write upgrade logic as parameter of queryFn
+  await queryFn(`DROP TABLE \`confirmed_transactions\`;`)
+  await queryFn(`DROP TABLE \`transaction_recipes\`;`)
+
+  await queryFn(`
+    ALTER TABLE \`accounts\` 
+      RENAME COLUMN \`balance\` TO \`balance_confirmed_at\`,
+      RENAME COLUMN \`balance_date\` TO \`balance_confirmed_at_date\`
+    ;
+  `)
+
+  await queryFn(
+    `ALTER TABLE \`accounts\` ADD COLUMN \`balance_created_at\` decimal(40,20) NOT NULL DEFAULT 0 AFTER \`balance_confirmed_at_date\`;`,
```

Comment:
```suggestion
    `ALTER TABLE \`accounts\` ADD COLUMN \`balance_on_creation\` decimal(40,20) NOT NULL DEFAULT 0 AFTER \`balance_confirmed_at\`;`,
```

the same here the balance-amount at the point of time on creation, right?

Reply:
correct, I like you suggestion, I update my code.